pkg/querier/queryrange/queryrangebase/roundtrip.go

```diff
@@ -64,7 +64,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.BoolVar(&cfg.ShardedQueries, "querier.parallelise-shardable-queries", true, "Perform query parallelisations based on storage sharding configuration and query ASTs. This feature is supported only by the chunks storage engine.")
 	f.Var(&cfg.ForwardHeaders, "frontend.forward-headers-list", "List of headers forwarded by the query Frontend to downstream querier.")
 
-	f.StringVar(&cfg.AcceptedQueryResponseFormat, "frontend.accepted-query-response-format", "json", "The downstream querier is requested to answer in the accepted format. Can be 'json' or 'protobuf'. Note: Both will still be routed over GRPC.")
+	f.StringVar(&cfg.RequestedQueryResponseFormat, "frontend.requested-query-response-format", "json", "The downstream querier is requested to answer in the accepted format. Can be 'json' or 'protobuf'. Note: Both will still be routed over GRPC.")
```

Comment:
Nit: I think "requested" here is a confusing name.
IMHO should be `required`; this is not a request that the querier can choose to ignore, right?